bin/GameIndex.dbf

```diff
@@ -9578,12 +9666,15 @@ Serial = SLES-51227
 Name   = Tomb Raider - Angel of Darkness
 Region = PAL-M10
 Compat = 5
-[patches]
-	// 54AD76D7
-	comment=- This gamedisc supports multiple languages and widescreen.
-	comment=- Language & widescreen selection is done through the BIOS configuration.
-	comment=- Implementation requires Full boot - "Boot CDVD (full)".
-	comment=- Full boot is recommended anyway for this gamedisc to avoid textproblems.
+[patches = 54AD76D7]
+	author=Prafull
+	// Fix ingame sps by interchanging vclipw.xyz vf5, vf5w with cfc2
+	// comment=- This gamedisc supports multiple languages and widescreen.
+	// comment=- Language & widescreen selection is done through the BIOS configuration.
+	// comment=- Implementation requires Full boot - "Boot CDVD (full)".
+	// comment=- Full boot is recommended anyway for this gamedisc to avoid textproblems. Needs DX11 to work properly
```

Comment:
These four comment lines are better placed at the top or at the bottom since they do not have anything to do with the fix, nor was it Prafull (but me) who pushed for these to be included here.

Reply:
Well i put at bottom, didnt know it was relevant or not.